### DISCUSSION TITLE
some types missing for intrinsic attributes

### DISCUSSION THREAD

**Discussion by trusktr** [solidjs/solid#365] [Ideas]
f.e.:

```
  Property 'onclick' does not exist on type 'ElementAttributes<Node, NodeAttributes, {}>'. Did you mean 'onClick'?ts(2322)
```

onclick works at runtime, but TypeScript sees only `onClick` as an available option. No biggie, easy to workaround. Would be sweet to add those types though, in the effort to match HTML.

**Comment by ryansolid** [comment]
Yeah you are right.. I added all lower and camelcase to other attributes but all the event handlers are missing. I should fix that. I feel like we had both at one point. I might have been trying to consolidate closer to React near the beginning.

**Reply by ryansolid** [reply]
Added in 0.24.14
